### PR TITLE
feat: show round / claim status on CLR page

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- add claim start and end data to Grant CLR
- uses that info to show info on if round is 
 - upcoming
 - active
 - ended
 - in claim phase
 
 This does not make any changes to the landing page on explorer as discussed with @frankchen07 as having a generic claim with diff periods does not scale
 
 

##### Refers/Fixes

https://gitcoin.atlassian.net/browse/GITC-466

##### Testing

https://giphy.com/gifs/FnZ0sZM0h4K1vf2Jso/fullscreen

### CODE REVIEW

**Review by ksolo — CHANGES_REQUESTED** [review]
Can you update the model tests to reflect the new properties?
https://github.com/gitcoinco/web/blob/master/app/grants/tests/models/test_grant_clr.py

**Comment by frankchen07** [comment]
@thelostone-mc this looks awesome!

1. Is there a way to change "Claim - Active ends {countdown}" when it's the claiming period? The statement doesn't really make sense. "Round - Ended" and "Round - Proposed" look good though.

2. How are you toggling which status appears? I'm a little confused on that. It looks like there's proposed, active, ended, claim active

cc @PixelantDesign this PR can handle the claim period for GR11 (since there's only one banner on the homepage for GR11), but for future CLR (side) rounds, we can choose to surface these claim dates either on the homepage or elsewhere in grant management. We can chat more about what a good solution here would look like.

**Comment by frankchen07** [comment]
reviewed the video on https://github.com/gitcoinco/web/pull/9549, left a couple of comments

**Review by chibie — CHANGES_REQUESTED** [review]

File: app/assets/v2/js/grants/index.js [hunk 1]
```diff
@@ -619,9 +496,67 @@ if (document.getElementById('grants-showcase')) {
...
+          end_date = moment(end_date).tz('America/Denver');
+
+          if (claim_start_date && claim_end_date) {
+            claim_start_date = moment(claim_start_date).tz('America/Denver');
+            claim_end_date = moment(claim_end_date).tz('America/Denver');
+          }
+
+          console.log(now);
+          console.log(claim_start_date);
+          console.log(claim_end_date);
```
[the remark this answers is not in the document]
Comment (chibie) [reply app/assets/v2/js/grants/index.js:521]: and some merge conflicts

**Review by ksolo — APPROVED** [review]

**Review by chibie — APPROVED** [review]

**Review by Jer-Sch — APPROVED** [review]

**Pull request merged**